### PR TITLE
fixes typo in components -> looping-through-lists  example

### PR DESCRIPTION
**Pull request by lifeart**
closes https://github.com/emberjs/ember.js/issues/19539

### CODE REVIEW

**Comment by locks** [comment]
@lifeart can you backport to relevant previous versions?

**Review by locks — APPROVED** [review]
Thanks!

File: guides/release/components/looping-through-lists.md [hunk 1]
```diff
@@ -405,7 +405,7 @@ export default class StoreCategoriesComponent extends Component {
     'Bourbons': ['Bulleit', 'Four Roses', 'Woodford Reserve'],
     'Ryes': ['WhistlePig', 'High West']
   };
-});
+};
```
Comment (ijlee2) [inline guides/release/components/looping-through-lists.md:408]:
```suggestion
}
```

**Review by ijlee2 — CHANGES_REQUESTED** [review]
I think `eslint` will remove the semicolon at the end of a class declaration. Before we merge the pull request, can we remove it?

**Review by ijlee2 — APPROVED** [review]
Thank you for helping with improving the documentation!

**Pull request merged**